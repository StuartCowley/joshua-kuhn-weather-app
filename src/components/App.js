import React from "react";
import LocationDetails from "./LocationDetails";
import "../styles/App.css";

function App({ location }) {
  const { city, country } = location;
  return <LocationDetails city={city} country={country} />;
}

export default App;
