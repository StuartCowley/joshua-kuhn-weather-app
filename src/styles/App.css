* {
  color: white;
}

.weather-app {
  margin: 1rem;
}
