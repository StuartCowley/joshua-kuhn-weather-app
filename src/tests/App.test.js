import React from "react";
import { render, screen } from "@testing-library/react";
import App from "../components/App";
import forecast from "../data/forecast.json";

test("renders learn react link", () => {
  render(<App location={forecast.location} forecasts={forecast.forecasts} />);
  const h1element = screen.getByText(/Manchester, UK/i);
  expect(h1element).toBeInTheDocument();
});
