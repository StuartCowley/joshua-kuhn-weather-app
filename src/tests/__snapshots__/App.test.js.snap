// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<DocumentFragment>
  <div
    class="weather-app"
  >
    <h1>
      , 
    </h1>
    <form
      class="search-form"
    >
      <input
        placeholder="Enter a city in the UK"
        type="text"
        value=""
      />
      <button
        type="submit"
      >
        Search
      </button>
    </form>
    <div
      class="forecast-summaries"
    />
  </div>
</DocumentFragment>
`;
